setup.py

```diff
@@ -0,0 +1,18 @@
+from setuptools import find_packages, setup
+
+
+setup(
+    name="ibm-fms-extras",
+    version="0.0.1",
+    author="Brian Vaughan, Joshua Rosenkranz, Antoni Viros i Martin, Davis Wertheimer, Supriyo Chakraborty, Raghu Kiran Ganti",
+    author_email="bvaughan@ibm.com, jmrosenk@us.ibm.com, aviros@ibm.com, Davis.Wertheimer@ibm.com, supriyo@us.ibm.com, rganti@us.ibm.com",
+    description="IBM Foundation Model Stack Extras",
+    url="https://github.com/foundation-model-stack/fms-extras",
+    packages=find_packages(),
+    install_requires=["torch >= 2.1", "ibm-fms >= 0.0.2", "transformers >= 4.35.0", "accelerate >= 0.26.1"],
```

Comment:
I think you want ibm-fms >= 0.0.3 since we're bumping the version?

Reply:
fixed in latest commit